src/cdk/platform/platform-module.ts

```diff
@@ -7,10 +7,7 @@
  */
 
 import {NgModule} from '@angular/core';
-import {Platform} from './platform';
 
 
-@NgModule({
-  providers: [Platform]
-})
+@NgModule()
```

Comment:
Out of curiosity, will we be able to kill this module off in the future? It appears to do nothing now

Reply:
I suppose so for now, though we could add other stuff to it. Maybe something like
`*cdkIfPlatform="IE11, Safari"`